.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+# Description
+A few sentences describing the overall goals of the pull request's commits.
+
+# Checks
```

Comment:
Not sure if "Checks" is the right word here. Maybe "Backports"? "Requirements"?

Reply:
I literally meant it to be "Checks". Things the person opening the PR has to check. Renaming it to "I have checked if ..." would be possible.